### PR TITLE
Add filter in manage_user

### PR DESCRIPTION
**Pull request by thainda-0252**
## Related Tickets
[- ticket redmine](https://edu-redmine.sun-asterisk.vn/issues/69401)

## WHAT (optional)
- Filter user for admin

## Evidence (Screenshot or Video)
image
image

### CODE REVIEW

**Comment by vinh-nd2002** [comment]
ok r

**Comment by thainda-0252** [comment]
ready

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -4,7 +4,13 @@ def new
   end
 
   def index
+    user = params[:name]
+    email = params[:email]
+    phone = params[:phone]
     @users = User.non_admin
+                 .by_user(user)
```
Comment (ngocvt-0484) [inline app/controllers/admin/users_controller.rb]: e gọi luôn params ở đây k cần gán biến

**Comment by thainda-0252** [comment]
ready

**Pull request merged**